src/LookingGlass.jl

```diff
@@ -235,5 +238,85 @@ module_recursive_globals_names(m::Module; constness=:all, mutability=:all) =
             if !isempty(names)
         ))
 
+"""
+    typetree(t::Type)
```

Comment:
Could we actually call this function `print_typetree` instead of `typetree`, to make it clear that it's not returning anything?

Reply:
Sure, that works. Extra clarity is probably worth a few keystrokes